server/index.js

```diff
@@ -45,6 +18,10 @@ const app = express();
 var port = normalizePort(process.env.PORT || '3000');
```

Comment:
process.env.PORT works perfectly for me without any additional packages. 
`PORT=3001 node index.js`  
 The docs you provided say the same thing. (no additional packages required.)
In the future it might be useful to use the nconf package tho. 
[https://www.npmjs.com/package/nconf](https://www.npmjs.com/package/nconf)

I think its a good practice to explicitly set the port to prevent errors. Making the default port 3000 will force one to explicitly set the port.
For the next commit I have created a bash script called `start_server.sh` which will start the server with the port 3001 and other environment variables.
the bash script can be started with `./start_server.sh`  This way it will be easier to start the server. 
I add it the next commit.